definitions/mlops-machine_learning_training/definition.yml

```diff
@@ -0,0 +1,19 @@
+domain: MLOPS
+type: MACHINE_LEARNING_TRAINING
+synthesis:
+  rules:
+    - identifier: aws..aws.sagemaker.trainingjobs.TrainingJobName
```

Comment:
mostly a doubt but is the job name unique? or we should use the ARN? (for both entities)

Reply:
discussed internally